packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/edit/translations/Messages_fr_FR.json

```
@@ -34,9 +34,9 @@
   "dedicatedCloud_USER_right_vmNetwork_noAccess": "Aucun accès",
   "dedicatedCloud_USER_right_vmNetwork_READONLY": "Accès en lecture seule",
   "dedicatedCloud_USER_right_vmNetwork_readonly": "Accès en lecture seule",
-  "dedicatedCloud_USER_right_vlan": "Accès au V(x)Lans",
-  "dedicatedCloud_USER_right_vlan_help_EPCC": "Droits de management sur la partie réseau privé (VxLan pour les Dedicated Cloud et VLAN pour les SDDC et les packs Hosted Private Cloud Premier)",
-  "dedicatedCloud_USER_right_vlan_help_MBM": "Droits de management sur la partie réseau privé (VxLan pour les Essentials et VLAN pour les SDDC et les packs Hosted Private Cloud Premier)",
+  "dedicatedCloud_USER_right_vlan_EPCC": "Accès au V(x)Lans",
```

@ganeshkumar1989 same thing here, this should be
```suggestion
  "dedicatedCloud_USER_right_vlan_EPCC": "Accès aux V(x)Lans",
```